atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -215,9 +215,8 @@ private static void createKeyspace(CassandraVerifierConfig verifierConfig) throw
     }
 
     private static boolean attemptToCreateKeyspace(CassandraVerifierConfig verifierConfig) {
-        Set<InetSocketAddress> thriftHosts = verifierConfig.servers().accept(new ThriftHostsExtractingVisitor());
-
-        return thriftHosts.stream().anyMatch(host -> attemptToCreateIfNotExists(host, verifierConfig));
+        return verifierConfig.servers().accept(ThriftHostsExtractingVisitor.INSTANCE).stream()
+                .anyMatch(host -> attemptToCreateIfNotExists(host, verifierConfig));
```

Comment:
no aversion whatsoever to this inlining, but is this a performance improvement or just a driveby? I'd be surprised if it was not the latter

Reply:
Yeah, this was a drive by in-line because the switch to visitor singleton enum made this wrap, so just inlined it